CreeDictionary/API/models.py

```diff
@@ -196,10 +201,19 @@ class Meta:
             models.Index(fields=["analysis"]),
             # text index benefits fast wordform matching (see search.py)
             models.Index(fields=["text"]),
+            # When we *just* want to lookup text wordforms that are "lemmas"
+            # (Note: Eddie thinks "head words" may also be lumped in as "lemmas")
+            # Used by:
+            #  - affix tree intialization
+            #  - sitemap generation
+            models.Index(fields=["is_lemma", "text"], name="lemma_text_idx"),
+            # TODO: do we need these indices?
             models.Index(fields=["inflectional_category"]),
             models.Index(fields=["pos"]),
         ]
 
+        ordering = ["is_lemma", "text"]
```

Comment:
I’m curious what the intent here is, as this change will add a global ordering. If sitemap generation needs a specific order, could the ordering be specified there instead?

Reply:
The reason I did this originally was because the sitemap generator [was emitting warnings](https://github.com/UAlbertaALTLab/cree-intelligent-dictionary/runs/2347894309#step:10:477) about an unordered object list. I thought I'd eliminate these warnings once and for all for every purpose, but now I'm realizing that this ordering is quite presumptuous. So I added an `.order_by()` clause on the queryset generated for sitemap generation instead! 👍🏼